docs/_sass/_main.scss

```diff
@@ -481,4 +503,38 @@ textarea {
         margin-left: auto;
         margin-right: auto;
     }
-}
\ No newline at end of file
+}
+
+@media screen and (max-width: 800px) {
+    #floating-concierge-button-lhn {
+        display: none;
+    }
+
+    .floating-concierge-button {
+        position: fixed;
+    }
+}
```

Comment:
This doesn't need to exist within a media query. It can be the default (code mobile-first), and then the `@media screen and (min-width: 801px)` breakpoint will override these values